tests/format/x/js/method-chain-with-inner-spacing/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -436,7 +435,34 @@ angular
 ================================================================================
 `;
 
-exports[`computed.js 1`] = `
+exports[`complex-args.js - {"arrowParens":"avoid","spaceInParens":true,"arrayBracketSpacing":true,"computedPropertySpacing":true,"spaceUnaryOps":true,"templateCurlySpacing":true,"typeAngleBracketSpacing":true} format 1`] = `
+====================================options=====================================
+arrayBracketSpacing: true
+arrowParens: "avoid"
+computedPropertySpacing: true
+parsers: ["babel", "babel-flow", "flow", "typescript"]
+printWidth: 80
+spaceInParens: true
+spaceUnaryOps: true
+templateCurlySpacing: true
+typeAngleBracketSpacing: true
+                                                                                | printWidth
+=====================================input======================================
+client.execute(
+  Post.selectAll()
+    .where(Post.id.eq(42))
+    .where(Post.published.eq(true))
+);
+
+=====================================output=====================================
+client.execute(
+  Post.selectAll().where( Post.id.eq( 42 ) ).where( Post.published.eq( true ) )
```

Comment:
I think this is a nice case from the existing Prettier test suite where the prettierX `--break-long-method-chains` option would give us better formatting.